NPPESAPI.net/NPPESAPI.net.csproj

```diff
@@ -1,7 +1,9 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;netstandard1.5;net451;net45</TargetFrameworks>
+    <TargetFrameworks>net5.0;netstandard2.1;netstandard2.0;netstandard1.5</TargetFrameworks>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <WarningsAsErrors />
```

Comment:
Are both of these needed? I don't quite remember how these work in tandem. I thought one overrides the other?

Reply:
`<WarningsAsErrors />` is the entry to specify any specific warning codes you'd like to ignore.  I usually clear it up front in case there are ever any defaults, then add on as needed if necessary